File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        int res =0, maxi =0;
        unordered_map<int,int> map;
        for (int num : nums){
            map[num]++;
            if (map[num] == maxi) res += maxi;
            else if (map[num] > maxi) {
                maxi = map[num];
                res = maxi;
            }
            else continue;
        }
        return res;
    }
};

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        c = Counter(nums)
        maxF, res = 0,0
        for num, freq in c.items():
            if freq > maxF:
                maxF = freq
                res = maxF
            elif freq == maxF:
                res += maxF
        return res